### PR TITLE
[Add-ins] (Yo Office guidance) Add notes about telemetry prompts and next steps guidance.

### PR DESCRIPTION
**Pull request by kbrandl**
This PR adds a couple of new notes to each article that contains instructions for using Yo Office -- a note about telemetry prompts, and a tip about ignoring 'next steps'. (Corresponding changes have been made in OfficeDev/office-js-docs-pr as part of this PR: https://github.com/OfficeDev/office-js-docs-pr/pull/1304.)

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[0ade05b](https://github.com/OfficeDev/outlook-dev-docs/commits/0ade05b90fc547fced5dadb30f8177523a8a4bdd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/addin-tutorial.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/add-ins/addin-tutorial.md) | :white_check_mark:Succeeded | |
[docs/add-ins/quick-start.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |
[docs/includes/yo-office-command-guidance.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/includes/yo-office-command-guidance.md) | :white_check_mark:Succeeded | |
[docs/includes/yo-office-next-steps.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/includes/yo-office-next-steps.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C17%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201909172220313554-704%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0WAN5N%2BJdmiLQ8Ijtjx5OSs0ycbDKczPnxoZIDkBaLw%3D&st=2019-09-17T22%3A16%3A28Z&se=2019-10-18T22%3A21%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/yo-office-command-guidance.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Run the following command to create an add-in project using the Yeoman generator: 
+
+```command&nbsp;line
+yo office
+```
+
+> [!NOTE]
+> When you run the `yo office` command, you may receive prompts about the data collection policies of Yeoman and the Office Add-in CLI tools. Use the information that's provided to respond to the prompts as you see fit. If you choose **Exit** in response to the second prompt, you'll need to run the `yo office` command again when you're ready to create your add-in project.
```
Comment (AlexJerabek) [inline docs/includes/yo-office-command-guidance.md]: It feels weird to specify the "second prompt" when they might not show up. Also, are we worried that people are going to be confused that exiting yo office ruins the tutorial? That seems intuitive to me.

Comment (kbrandl) [reply docs/includes/yo-office-command-guidance.md]: @AlexJerabek Valid points -- thanks for the feedback! I'll remove the last sentence from this note (starting with "If you choose Exit..."), and will make the same change to this verbiage in OfficeDev/office-js-docs-pr.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[b9ea43e](https://github.com/OfficeDev/outlook-dev-docs/commits/b9ea43edba280026f72bd8495539590977f0f536)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/addin-tutorial.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/add-ins/addin-tutorial.md) | :white_check_mark:Succeeded | |
[docs/add-ins/quick-start.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/add-ins/quick-start.md) | :white_check_mark:Succeeded | |
[docs/includes/yo-office-command-guidance.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/includes/yo-office-command-guidance.md) | :white_check_mark:Succeeded | |
[docs/includes/yo-office-next-steps.md](https://github.com/OfficeDev/outlook-dev-docs/blob/kb-yo-office-guidance-updates/docs/includes/yo-office-next-steps.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C18%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201909181744438949-704%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=I2CYkowqSBLBDxmfjDRXECMQXxVTe%2BV5jJw%2FYJqaoTQ%3D&st=2019-09-18T17%3A40%3A49Z&se=2019-10-19T17%3A45%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**